06week/higherOrder.js

```diff
@@ -3,15 +3,46 @@
 const assert = require('assert');
 
 function forEach(arr, callback) {
-  // Your code here
+  // Create a forEach() function that takes an array of items and a function that runs the function arr.length number of times.
+  const loopThrough = (arr, callback) => {
```

Comment:
This isn't ever called in your function, yielding no return from the actual loop.  You could omit the loopThrough function completely and it would yield the expected results for a forEach higher order function